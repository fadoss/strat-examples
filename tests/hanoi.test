games/hanoi

srew initial(4, 3) using moveAll(0, 2, 4) .
