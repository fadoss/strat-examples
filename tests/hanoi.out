==========================================
srewrite in HANOI-SOLVE : initial(4, 3) using moveAll(0, 2, 4) .

Solution 1
rewrites: 62
result Hanoi: (0)[nil] (1)[nil] (2)[4 3 2 1]

No more solutions.
rewrites: 62
Bye.
